tests/iam/identhandler/fileidentifier_test.cpp

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "aos/common/tools/buffer.hpp"
+#include "mocks/subjectsobservermock.hpp"
+#include <gtest/gtest.h>
+
+#include "aos/iam/identhandler/fileidentifier.hpp"
+
+using namespace aos;
+using namespace aos::iam::identhandler;
+using namespace testing;
+
+/***********************************************************************************************************************
+ * Suite
+ **********************************************************************************************************************/
+
+class FileIdentifierTest : public Test {
+protected:
```

Comment:
Only SetUp and TearDown method should be protected consts and members could be private?
Group items in class as following:

consts
methods
vars

Reply:
Re-grouped items, but kept them protected, `cause they are used in the UT(s).